unstructured_inference/models/yolox_model.py

```diff
@@ -0,0 +1,130 @@
+import os
+import tempfile
+
+import cv2
+import jsons
+import numpy as np
+import onnxruntime
+from pdf2image import convert_from_path
+
+import unstructured_inference
+from unstructured_inference.inference.layout import DocumentLayout, LayoutElement, PageLayout
+from unstructured_inference.visualize import vis
+from unstructured_inference.yolox_functions import demo_postprocess, multiclass_nms
+from unstructured_inference.yolox_functions import preproc as preprocess
+from unstructured_inference.models import _get_model_loading_info
+
+
+def yolox_local_inference(
+    filename: str, type: str = "image", to_json: bool = False, keep_output: bool = False
+):
+    """This function creates a DocumentLayout from a file in local storage.
```

Comment:
Could you format this doc string similar to this one?

- https://github.com/Unstructured-IO/unstructured/blob/8abf1f119d5439003beeed65fafdcc447ad839b9/unstructured/partition/image.py#L14-L29